libEDSsharp/CanOpenXDD.cs

```
@@ -1184,12 +1184,11 @@ public EDSsharp convert(ISO15745ProfileContainer container)
                                             eds.fi.EDSVersion = keyvalue[1];
                                             break;
                                         case "FileRevision":
-                                            eds.fi.FileVersion = keyvalue[1];
+                                            byte.TryParse(keyvalue[1], out eds.fi.FileVersion);
```

silently ignoring failed parsing